api-gen/rendering/templates/class-member-list.tsx

```diff
@@ -0,0 +1,12 @@
+/** @jsx h */
+import {h} from 'preact';
```

Comment:
Just a drive by comment, these imports weren't necessary when I was playing around with it, though I did need an otherwise unused dependency on `//:node_modules/preact`. I suspect these two lines can be dropped with the right configuration?

Example: https://github.com/dgp1130/rules_prerender/blob/85bfe6cf5a2cf42fbed1c842dba5c0cccabbe306/docs/components/header/header.tsx
`tsconfig.json`: https://github.com/dgp1130/rules_prerender/blob/85bfe6cf5a2cf42fbed1c842dba5c0cccabbe306/tsconfig.json#L13-L14

Of course I'm not that familiar with Preact/JSX either, so it's possible I'm doing something wrong.

Reply:
The `/** @jsx h */` commented ended up not being necessary, but the `import` is definitely load-bearing. I think it _wouldn't_ be necessary if the `jsxImportSource` setting worked, but there's some incompatibility there with this older `ts_library` implementation.